akka-contrib/src/main/scala/akka/contrib/circuitbreaker/CircuitBreakerActor.scala

```diff
@@ -0,0 +1,247 @@
+/**
+ * Copyright (C) 2014-2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.contrib.circuitbreaker
+
+import akka.actor._
+import akka.event.LoggingAdapter
+import akka.pattern._
+import akka.util.Timeout
+
+import scala.concurrent.{ ExecutionContext, Future }
+import scala.util.{ Failure, Success }
+
+object CircuitBreakerActor {
```

Comment:
Perhaps we should also add a doc here that "This is an Actor which implements the circuit breaker pattern, you may also be interested in the raw circuit breaker [[akka.pattern.CircuitBreaker]]"?